src/backend/src/utils/slack.utils.ts

```diff
@@ -43,16 +45,27 @@ export const sendSlackUpcomingDeadlineNotification = async (workPackage: WorkPac
 
 /**
  * Send CR requested review notification to reviewer in Slack
- * @param slackId the slack id of the reviewer
+ * @param slackIds the slack ids of the reviewers
  * @param changeRequest the requested change request to be reviewed
  */
-export const sendSlackRequestedReviewNotification = async (slackId: string, changeRequest: ChangeRequest): Promise<void> => {
+export const sendSlackRequestedReviewNotification = async (
+  reviewers: UserWithSettings[],
+  changeRequest: ChangeRequest,
+  threads: {
+    messageInfoId: string;
+    channelId: string;
+    timestamp: string;
+    changeRequestId: number;
+  }[]
+): Promise<void> => {
   if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
 
+  const btnText = `View CR#${changeRequest.crId}`;
   const changeRequestLink = `<https://finishlinebyner.com/change-requests/${changeRequest.crId.toString()}>`;
+  const fullMsg =
+    usersToSlackPings(reviewers) + `Your review has been requested on CR #${changeRequest.crId}: ${changeRequestLink}.`;
 
-  const fullMsg = `Your review has been requested on CR #${changeRequest.crId}: ${changeRequestLink}.`;
-  await sendMessage(slackId, fullMsg);
+  threads.map((thread) => replyToMessageInThread(thread.channelId, thread.timestamp, fullMsg, changeRequestLink, btnText));
```

Comment:
We wanna use `forEach` instead of `map` here to run void function on every thread element